site-book/README.md

```diff
@@ -0,0 +1,51 @@
+# Site Book Metron documentation 
+
+Metron's Site Book is an attempt at producing documentation that is
+
+- Versioned and reviewed,
+- Only committers can edit
```

Comment:
I would recommend deleting "only committers can edit".  It is actually self-evident, and seems unnecessary to state as a primary objective.  The other three are tremendously more important.